card/mod/bootstrap/set/abstract/bootstrap_code_file.rb

```diff
@@ -1,42 +1,45 @@
+
 def self.included host_class
   host_class.include_set Abstract::CodeFile
+  host_class.include OverrideCodeFile
   host_class.mattr_accessor :stylesheets_dir, :bootstrap_path, :mod_path
   host_class.mod_path = File.join Cardio.gem_root, "mod", host_class.file_content_mod_name
   host_class.stylesheets_dir = File.join host_class.mod_path, "lib", "stylesheets"
   host_class.bootstrap_path = File.join host_class.mod_path, "vendor", "bootstrap", "scss"
 end
```

Comment:
at some point we may want to abstract the parts of this that are just "CodeMultiFile".   Not that much of this code is really bootstrap specific.

Reply:
CodeFile already handles multiple files (all you have to do is return an array of filenames in `source_files`).
All the code here is about special paths to (bootstrap) stylesheet directories.